server/v2/cometbft/abci.go

```diff
@@ -153,10 +161,9 @@ func (c *Consensus[T]) Info(ctx context.Context, _ *abciproto.InfoRequest) (*abc
 	}
 
 	return &abciproto.InfoResponse{
-		Data:    c.cfg.Name,
-		Version: c.cfg.Version,
-		// AppVersion:       cp.GetVersion().App,
-		AppVersion:       0, // TODO fetch from store?
+		Data:             c.appName,
+		Version:          c.version,
+		AppVersion:       0, // TODO fetch consensus params?
```

Comment:
yes we should fetch this.

Reply:
We should enable in a follow-up, as it fails getting them:

`failed to start servers: error during handshake: error calling Info: rpc error: code = Internal desc = collections: not found: key 'no_key' of type github.com/cosmos/gogoproto/cometbft.types.v1.ConsensusParams`

Reply:
https://github.com/cosmos/cosmos-sdk/issues/20996